scholarslab/scholarslab.org
# Incorporate prototype of blog post layout/design

# Description
- Creates layout for blog posts, and any other pages w/ layout `post.html`

## Related Ticket(s)
- progress on #119, need to do #133, closes #137 
- progress on #6 
- still need to address #36, #39, #147, #148, #150

----
#### Problem to Solve 
- Lack of stylesheet for blog posts, which make up the bulk of our rendered HTML.

#### Expected Behavior 
- All blog posts can successfully load without glitching, repeating elements, breaking the document flow, etc.
- Posts are both pleasing to look at and easy to read.

#### Steps to Test Solution 

- Load a whole bunch of posts from different years - I know that some of the most recent ones have some wonky Markdown issues. (see issues referenced above)
